.github/workflows/release-ci.yml

```diff
@@ -0,0 +1,87 @@
+name: npm - release to dev/integration
+run-name: npm - release from ${{ github.ref_name }} to dev/integration
+
+on:
+  push:
+    branches:
+      - main
+      - 'integration/*'
+      - '*.*.x'
+    paths-ignore:
+      - '*.md'
+      - '.devcontainer/**'
+      - '.vscode/**'
+      - '.github/ISSUE_TEMPLATE/**'
+      - 'LICENSE'
+      - '.github/CODEOWNERS'
+      - '.buildkite/**'
+      - '.dockerignore'
+      - '*.bench.ts'
+      - 'graphs/**'
+  workflow_dispatch:
+
+env:
+  # To hide "Update available 0.0.0 -> x.x.x"
+  - PRISMA_HIDE_UPDATE_MESSAGE="true"
+
+jobs:
+  release:
+    timeout-minutes: 60
+    environment: 'npm-ci'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install & build
+        uses: ./.github/actions/setup
+        with:
+          node-version: 16
+
+      - name: Publish all packages to npm
+        run: pnpm run publish-all
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+          BUILDKITE: true
+          BUILDKITE_BRANCH: ${{ github.ref_name }}
+          GITHUB_CONTEXT: ${{ toJson(github) }}
+          # Note: must use personal access token
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          REDIS_URL: ${{ secrets.REDIS_URL }}
+
+  # Cuurently done in `sendSlackMessage()` in publish.ts, we should move it here later
+  # success:
+  #   needs:
+  #     - release
+  #   if: ${{ success() }}
+  #   name: Communicate success
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #     - name: Set current job url in SLACK_FOOTER env var
+  #       run: echo "SLACK_FOOTER=<$GITHUB_SERVER_URL/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID|Click here to go to the job logs>" >> $GITHUB_ENV
+
+  #     - name: Slack Notification on Success
+  #       uses: rtCamp/action-slack-notify@v2.2.0
+  #       env:
+  #         SLACK_TITLE: 'prisma/prisma Release failed :x:'
+  #         SLACK_COLOR: '#FF0000'
+  #         SLACK_WEBHOOK: ${{ secrets.SLACK_RELEASE_FEED_WEBHOOK }}
+  #         SLACK_CHANNEL: feed-prisma-releases
+
+  failure:
```

Comment:
iiuc, this is more easily composable than adding a step in each job?

Reply:
I changed some things here, added success to mimic what we have in Buildkite and kept `sendSlackMessage` which is different.